v3/bind.go

```diff
@@ -733,3 +733,115 @@ RESP:
 
 	return nil, GetLDAPError(packet)
 }
+
+// ServerBindStep sends the SASLBindRequest and return the result code, servercred and errors.
```

Comment:
Can you please add a short example on how to use this function to our tests? Thank you!